tools/sense_studio/templates/navigation.html

```diff
@@ -52,13 +52,13 @@
         </div>
     </div>
 </div>
-<a href="#" class="item">Train Model</a>
-<a href="#" class="item">Evaluate Model</a>
+<a href="javascript:void(0)" class="item hashoverpopup" data-content="Coming Soon! Hint: Use train_classifier.py" data-position="bottom left" data-variation="basic">Training</a>
```

Comment:
Tell me if `data-content` need to be more informative, right now the content is basic hint.